design-proposals/passt-feature-gate-iface-api-deprecation.md

```diff
@@ -0,0 +1,66 @@
+## Overview
+Following the emerging [Kubevirt Network Binding Plugin API](https://docs.google.com/document/d/13fLaqZuM8El_CT5XzxPv-1epKsB1-Spv_jtXveEkkdQ/edit#heading=h.z27rudfpwy44)
+and introduction of Passt network binding plugin [2](https://github.com/kubevirt/kubevirt/pull/10425) [3](https://github.com/kubevirt/kubevirt/pull/10621), 
+it's now possible to remove Passt network binding implementation from Kubevirt core.
+
+This doc outlines deprecation plan for Passt network binding, interface API and  feature gate.
+
+# Motivation
+Removing Passt network binding implementation from Kubevirt core and use Passt network binding plugin instead, 
```

Comment:
Can you please start with the reason (maintenance) and mention the "how" (replace) in the "Design" section?

Reply:
Done